.gitignore

```diff
@@ -27,3 +27,5 @@
 /docker-dev.lock
 node_modules
 /web/apps/atelier-des-idees/src/App.css
+
+\.DS_Store
```

Comment:
you need to ignore this in your global gitignore IMO

Reply:
done, thanks !